### PR TITLE
Adapt 'validator status' presentation

### PR DESCRIPTION
**Pull request by marcel-bitfly**
New 'types` were added, some where renamend.
When including 'online` they should appear green.
When including 'offline` they should appear red.
Otherwise orange.

See: BIDS-3218

### CODE REVIEW

**Comment by marcel-bitfly** [comment]
is this an issue?
image

**Review by D13ce — CHANGES_REQUESTED** [review]

File: frontend/components/validator/table/ValidatorTableStatus.vue
[the remark this answers is not in the document]
Comment (marcel-bitfly) [reply frontend/components/validator/table/ValidatorTableStatus.vue:1]: @LuccaBitfly could the problem be on the `backend`?

[the remark this answers is not in the document]
Comment (LuccaBitfly) [reply frontend/components/validator/table/ValidatorTableStatus.vue:1]: @marcel-bitfly Yes the new states still need to be handled correctly in data access.

**Review by D13ce — APPROVED** [review]
As discussed, frontend changes seem to be fine.

**Pull request merged**